hnn_core/cell.py

```diff
@@ -232,15 +227,16 @@ class Cell:
     }
     """
 
-    def __init__(self, name, pos, p_secs, p_syn, topology, sect_loc, gid=None):
+    def __init__(self, name, pos, sections, synapses, topology, sect_loc,
+                 gid=None):
         self.name = name
         self.pos = pos
-        self.p_secs = p_secs
-        self.p_syn = p_syn
+        self.sections = sections
+        self.synapses = synapses
         self.topology = topology
         self.sect_loc = sect_loc
-        self.sections = dict()
-        self.synapses = dict()
+        self._nrn_sections = dict()
+        self._nrn_synapses = dict()
```

Comment:
Good call. This helps us more readily know exactly where Neuron objects are stored.